### ISSUE TITLE
A11y - Zoom-Links in Kalendereinträgen sind nicht eindeutig identifizierbar

### ISSUE THREAD

**Issue by lukasniebler** [RRZE-Webteam/FAU-Einrichtungen#1130]
Die Zoom-Links in Terminübersichten sind nicht als Links erkennbar. Es fehlt die Farbe + Unterstreichung s. Siteimprove. FAU Einrichtungen-Theme

https://my2.siteimprove.com/Inspector/1098529/A11Y/Page?pageId=91709153450&impmd=3DF0A3E50EC9CF0E4EE290ED53EF8D10#/sia-r62/failed/Y1BXRTFreDI=;YzE0ZjkzNTI=

**Comment by xwolfde** [comment]
Wird in der 2.1 Release behoben sein
